apps/aklite-offline/cmds.cpp

```diff
@@ -12,6 +12,16 @@ int InstallCmd::installUpdate(const Config& cfg_in, const boost::filesystem::pat
         offline::client::install(cfg_in, {src_dir / "tuf", src_dir / "ostree_repo", src_dir / "apps"});
 
     switch (install_res) {
+      case offline::PostInstallAction::NeedRebootForBootFw: {
```

Comment:
Is this a necessary step if we do it in the pre-update stage.

Reply:
It's preferable. The normal update flow is:

1. `install -s <>`
2. reboot
3. `run` (finalize ostree/rootfs update), if exit code is `90` then step 4.
4. reboot to confirm booting on the new ostree/rootfs for the bootloader

We do the pre-update stage to handle just in case if the previous update is not fully completed or `bootupgrade_available` is set by mistake.